src/stdlib/builtins.md

```diff
@@ -147,77 +147,182 @@ The following generic built-ins compile to WebAssembly instructions directly.
+
+  | T                       | Instruction
+  |-------------------------|-------------
+  | f32                     | f32.abs
+  | f64                     | f64.abs
+  | i8, i16, i32, i64       | *emulated*
+  | u8, u16, u32, u64, bool | *omitted*
+  </details>
 
 * ```ts
   function max<T>(left: T, right: T): T
   ```
-  Determines the maximum of two integers or floats. If either operand is `NaN`, returns `NaN`.
+  <details><summary>Determines the maximum of two integers or floats. If either operand is <code>NaN</code>, returns <code>NaN</code>.</summary>
+
+  | T                                          | Instruction
+  |--------------------------------------------|-------------
+  | f32                                        | f32.max
+  | f64                                        | f64.max
+  | i8, u8, i16, u16, i32, u32, i64, u64, bool | *emulated*
+  </details>
 
 * ```ts
   function min<T>(left: T, right: T): T
   ```
-  Determines the minimum of two integers or floats. If either operand is `NaN`, returns `NaN`.
+  <details><summary>Determines the minimum of two integers or floats. If either operand is <code>NaN</code>, returns <code>NaN</code>.</summary>
+
+  | T                                          | Instruction
+  |--------------------------------------------|-------------
+  | f32                                        | f32.min
+  | f64                                        | f64.min
+  | i8, u8, i16, u16, i32, u32, i64, u64, bool | *emulated*
+  </details>
 
 * ```ts
   function ceil<T>(value: T): T
   ```
-  Performs the ceiling operation on a 32-bit or 64-bit float.
+  <details><summary>Performs the ceiling operation on a 32-bit or 64-bit float.</summary>
+
+  | T                                          | Instruction
+  |--------------------------------------------|-------------
+  | f32                                        | f32.ceil
+  | f64                                        | f64.ceil
+  | i8, u8, i16, u16, i32, u32, i64, u64, bool | *omitted*
+  </details>
 
 * ```ts
   function floor<T>(value: T): T
   ```
-  Performs the floor operation on a 32-bit or 64-bit float.
+  <details><summary>Performs the floor operation on a 32-bit or 64-bit float.</summary>
+
+  | T                                          | Instruction
+  |--------------------------------------------|-------------
+  | f32                                        | f32.floor
+  | f64                                        | f64.floor
+  | i8, u8, i16, u16, i32, u32, i64, u64, bool | *omitted*
+  </details>
 
 * ```ts
   function copysign<T>(x: T , y: T): T
   ```
-  Composes a 32-bit or 64-bit float from the magnitude of `x` and the sign of `y`.
+  <details><summary>Composes a 32-bit or 64-bit float from the magnitude of <code>x</code> and the sign of <code>y</code>.</summary>
+
+  | T   | Instruction
+  |-----|-------------
+  | f32 | f32.copysign
+  | f64 | f64.copysign
```

Comment:
I guess make sense add this:
```ts
| i8, u8, i16, u16, i32, u32, i64, u64, bool | *omitted*
```
for consistency

Reply:
This is what the compiler does, there are no fillers for integers here. Would you prefer to have fillers here?

Reply:
For context, `omitted` above means that the operation accepts the type argument, but does not emit an instruction and just pipes through.

Reply:
Ah wait, isn't `*omitted*` means unsupported (compile error)?